worlds/lingo/player_logic.py

```diff
@@ -463,6 +467,9 @@ def calculate_panel_requirements(self, room: str, panel: str, world: "LingoWorld
                                                                     req_panel.panel, world)
                 access_reqs.merge(sub_access_reqs)
 
+            if self.victory_condition == (panel_object.location_name or f"{room} - {panel}"):
```

Comment:
Uhhh
I don't think this makes any sense

Reply:
I am now somewhat baffled that this PR seems to work correctly

Reply:
As discussed offline, it turns out my brain is being studied in a lab.